src/main/play-doc/general/MigrationGuide.md

```diff
@@ -33,18 +33,325 @@ In additional to disabling Elasticsearch via configuration, a service lookup mus
   -Dcontrail.syslog.server.service-locator.enabled=off
 ```
 
-### Service ports
+## Service endpoint declaration
 
-**The services is now deprecated in lieu of [request acl](RequestAclConfiguration).**
+The services endpoint declaration defines the protocol, port, and/or path under which your service will be addressed to the outside world.
 
-The services define the protocol, port, and/or path under which your service will be addressed to the outside world on. For example, if http and port 80 are to be used to provide your services and then the following expression can be used to resolve `/myservice` on:
+**From 1.2 onwards, the service ports are deprecated in lieu of [[request ACL|AclConfiguration]].**
```

Comment:
'service endpoints are deprecated' vs 'service ports' to be consistent with header